ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -238,3 +238,29 @@ def test_tensorflow_negative(
         fn_tree="negative",
         x=np.asarray(x, dtype=input_dtype),
     )
+
+
+# log_sigmoid
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_tf.valid_float_dtypes),
```

Comment:
You'll probably have to add in the min and max safety factor parameters (use 1.0 as their values) in order to solve this error